test/source-updater.test.js

```diff
@@ -810,7 +810,7 @@ function(assert) {
 
 QUnit.test('setDuration blocks audio and video queue entries until it finishes',
 function(assert) {
-  const done = assert.async();
+  const done = assert.async(2);
```

Comment:
wait for two `done` calls, one from the video appendBuffer and one from the audio appendBuffer.

Reply:
That's a thing? Neat.